HW4/Greenhouse.java

```diff
@@ -0,0 +1,41 @@
+package HW4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Greenhouse {
```

Comment:
missed new line
do Ctrl+Shift+L in each class

Reply:
still no new line here

Reply:
updated